installing/installing_openstack/installing-openstack-user.adoc

```diff
@@ -45,7 +46,11 @@ include::modules/installation-initializing.adoc[leveloffset=+1]
 
 include::modules/installation-osp-custom-subnet.adoc[leveloffset=+2]
 include::modules/installation-osp-config-yaml.adoc[leveloffset=+2]
+//Dual-stack networking
+include::modules/install-osp-dualstack.adoc[leveloffset=+2]
+include::modules/install-osp-deploy-dualstack.adoc[leveloffset=+3]
 include::modules/installation-osp-fixing-subnet.adoc[leveloffset=+2]
+include::modules/installation-osp-configuring-machine-network-cidr.adoc[leveloffset=+2]
```

Comment:
looks like this docs is a combination of `installation-osp-custom-subnet.adoc` and `installation-osp-fixing-subnet.adoc`, which makes them unnecessary. 

@gryf @mandre  WDYT?

Reply:
@gryf @mandre please advise :)

Reply:
I don't see exactly how `modules/installation-osp-configuring-machine-network-cidr.adoc` is different from `modules/installation-osp-fixing-subnet.adoc`. If there are any changes needed, they should probably go there.

Reply:
`installation-osp-custom-subnet.adoc` is fine AFAIU

Reply:
@obrown1205 I agree with Martin, `installation-osp-fixing-subnet.adoc` has the same goal of `installation-osp-configuring-machine-network-cidr.adoc` but the latter is an automated way, so let's remove the former.

Reply:
I still don't see how `installation-osp-fixing-subnet.adoc` and `installation-osp-configuring-machine-network-cidr.adoc` differ. We need to consolidate them in a single module.

`installation-osp-fixing-subnet.adoc` is imported from `installing/installing_openstack/installing-openstack-user-sr-iov.adoc` as well.